rules/azure-budgets/rule.md

```diff
@@ -127,7 +129,7 @@ ${{ AMOUNT }} AUD + GST/year
 
 1. Please approve
```

Comment:
The Issue: 
```
1. Please approve

\< As per https://ssw.com.au/rules/azure-budgets \>
:::
:::

3. If the request is approved, remember to add a cost-category tag to the new resource once it is created
```

The issue is being picked up due to the above ordered list being out of sync. Since it doesn't know that the 1 is part of an email and not a continuation of the 2 further up.